build-push-image/ecr_create.sh

```diff
@@ -1,45 +1,69 @@
 #!/bin/bash
 set -e
 
-url_encoded_ecr_repository=$(echo "$ECR_REPOSITORY" | sed 's/\//\%2F/g')
-
-echo "::debug::ECR_REPOSITORY: $ECR_REPOSITORY"
-echo "::debug::URL_ECR_REPOSITORY: $url_encoded_ecr_repository"
+# Extract the branch name from GITHUB_REF
+if [[ "$GitVersion_PreReleaseLabel" != "" ]]; then
+    LIFECYCLE_POLICY="{
+        \"rules\": [
+            {
+                \"rulePriority\": 1,
+                \"selection\": {
+                    \"tagStatus\": \"any\",
+                    \"countType\": \"imageCountMoreThan\",
+                    \"countNumber\": 30
+                },
+                \"action\": {
+                    \"type\": \"expire\"
+                }
+            }
+        ]
+        }"
```

Comment:
Everything is the same except the countNumber? Can you refactor this code?

Reply:
yes on it

Reply:
Done